### ISSUE TITLE
what is the role of tf.squeeze for DDPG code in tensorflow ? 

### ISSUE THREAD

**Issue by johnnylin110** [openai/spinningup#339]
Hello, 
Thanks for providing the DDPG code in tensorflow 1 version,  and I also try to implement by my own way.
However, I see that in your core function for DDPG, there is a tf.squeeze for q and q_pi 
[https://github.com/openai/spinningup/blob/master/spinup/algos/tf1/ddpg/core.py#L33](url) . 
Why we need the tf.squeeze here?  And also, when I move the tf.squeeze away, the performance become bad for different seeds. 
In my opinion, i think it will return the shape  (batchsize,) instead of (batchsize,1) for the q value function.  And I think the two shape didn't affect the training when we update by `q_loss = tf.reduce_mean((q-backup)**2)` ( the code in [https://github.com/openai/spinningup/blob/master/spinup/algos/tf1/ddpg/ddpg.py](url).)
However, it did affect the performance, with the squeeze , the code can get 130 up reward in Swimmer-v2. Without the Squeeze, it only can reach from 10 to 50 and variance bigger. 

Thanks for helping
